module/VuFind/config/module.config.php

```diff
@@ -346,17 +346,18 @@
             'VuFind\Favorites\FavoritesService' => 'VuFind\Favorites\FavoritesServiceFactory',
             'VuFind\GeoFeatures\BasemapConfig' => 'VuFind\GeoFeatures\AbstractConfigFactory',
             'VuFind\GeoFeatures\MapTabConfig' => 'VuFind\GeoFeatures\AbstractConfigFactory',
-            'VuFind\GeoFeatures\MapSelectionConfig' => 'VuFind\GeoFeatures\AbstractConfigFactory',
-            'VuFind\Hierarchy\Driver\PluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
+            'VuFind\GeoFeatures\MapSelectionConfig'            => 'VuFind\GeoFeatures\AbstractConfigFactory',
+            'VuFind\Hierarchy\Driver\PluginManager'            => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
             'VuFind\Hierarchy\TreeDataFormatter\PluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
-            'VuFind\Hierarchy\TreeDataSource\PluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
-            'VuFind\Hierarchy\TreeRenderer\PluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
-            'VuFind\ILS\Connection' => 'VuFind\ILS\ConnectionFactory',
-            'VuFind\ILS\Driver\PluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
-            'VuFind\ILS\Logic\Holds' => 'VuFind\ILS\Logic\LogicFactory',
-            'VuFind\ILS\Logic\TitleHolds' => 'VuFind\ILS\Logic\LogicFactory',
-            'VuFind\ILS\HoldSettings' => 'VuFind\ILS\HoldSettingsFactory',
-            'VuFind\Log\Logger' => 'VuFind\Log\LoggerFactory',
+            'VuFind\Hierarchy\TreeDataSource\PluginManager'    => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
+            'VuFind\Hierarchy\TreeRenderer\PluginManager'      => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
+            'VuFind\I18n\Initializer'                          => 'VuFind\I18n\InitializerFactory',
+            'VuFind\ILS\Connection'                            => 'VuFind\ILS\ConnectionFactory',
+            'VuFind\ILS\Driver\PluginManager'                  => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
+            'VuFind\ILS\Logic\Holds'                           => 'VuFind\ILS\Logic\LogicFactory',
+            'VuFind\ILS\Logic\TitleHolds'                      => 'VuFind\ILS\Logic\LogicFactory',
+            'VuFind\ILS\HoldSettings'                          => 'VuFind\ILS\HoldSettingsFactory',
+            'VuFind\Log\Logger'                                => 'VuFind\Log\LoggerFactory',
```

Comment:
Is there a reason you aligned all of these operators? It creates a seemingly irrelevant diff in the PR. I understand that there are some advantages to aligning them, but we've generally settled on the convention of not aligning them, and it just seems unusual to have a small patch of aligned operators here. Did your editor just do this of its own accord?

Reply:
> Did your editor just do this of its own accord?
Sorry, next time I'll check for such things before pushing. And yes, my editors beautifier is responsible for the alignment. I will modify the PR in order to stick to the convention.